test/posexional/field/field_test.exs

```diff
@@ -0,0 +1,214 @@
+defmodule Posexional.Field.FieldTest do
+  use Posexional.Case, async: true
+
+  defmodule Row1 do
+    use PosexionalRow, [:as_struct]
+
+    guesser &__MODULE__.guesser/1
+
+    fixed_value("row1")
```

Comment:
does it make sense to avoid the use of parentheses as for ecto schema, since it is a macro?

Reply:
sorry it is outdated, I will resolve it